feat: deprecate personal Vercel id for team id

phidol opened the pull request:
See #28

paulogdm reviewed (approved):
Looks good by the code, but can we test this?
Also, double check `task.json` values.

phidol commented:
Hey @paulogdm,
I've rebased the PR and adjusted a spot I've missed. Unfortunately the change needs to be merged to create a preview.
I can setup an example of a 'dysfunctional' config as well as the already documented proper configuration in a public Azure project of mine as soon as there's a preview version available to verify the changes!

phidol commented:
Hey @paulogdm,
I'm neither allowed to merge the changes nor to run the pipeline for the creation of the preview version.
Could you please do so?

Zertsov commented:
@phidol Hey! I just pinged the team about this PR, I'll update when I hear back

Zertsov commented:
@phidol I'll make sure a preview release is cut today!
